Compiling SWIG-generated wrappers with `clang` on Windows fails

```
"clang++.cmd" "-Iexample.node.p" "-I." "-I..\.." "-I..\..\node_modules\node-addon-api" "-I..\..\src" "-fcolor-diagnostics" "-D_FILE_OFFSET_BITS=64" "-Wall" "-Winvalid-pch" "-Wnon-virtual-dtor" "-std=c++17" "-O0" "-g" "-isystemC:\Users\mmom\AppData\Local\node-gyp\Cache\18.19.0\include\node" "-fpermissive" -MD -MQ example.node.p/build_example_wrap.cc.obj -MF "example.node.p\build_example_wrap.cc.obj.d" -o example.node.p/build_example_wrap.cc.obj "-c" ../../build/example_wrap.cc
../../build/example_wrap.cc:3210:12: error: cast from pointer to smaller type 'long' loses information
 3210 |   result = (long)arg1;
      |            ^~~~~~~~~~
1 error generated.
```

--- a/CHANGES-JSE
+++ b/CHANGES-JSE
@@ -7,6 +7,10 @@ the issue number to the end of the URL: https://github.com/swig/swig/issues/
 Version 5.0.5
 ===========================
 
+2024-09-08: mmomtchev
+            [JavaScript] Fix [mmomtchev/swig#37](https://github.com/mmomtchev/swig/issues/37),
+            Generated code compilation fails with `clang` on Windows
+
 2024-09-01: mmomtchev
             [JavaScript] Fix [mmomtchev/swig#62](https://github.com/mmomtchev/swig/issues/62),
             eliminate `$argnum` from JavaScript C-style array typemaps

--- a/Lib/javascript/napi/javascriptrunbody.swg
+++ b/Lib/javascript/napi/javascriptrunbody.swg
@@ -343,29 +343,6 @@ fail:
 #endif
 }
 
-SWIGRUNTIME Napi::Value _wrap_getCPtr(const Napi::CallbackInfo &info) {
-  Napi::Env env = info.Env();
-  Napi::Value jsresult;
-  void *arg1 = (void *) 0 ;
-  long result;
-  int res1;
-
-  res1 = SWIG_GetInstancePtr(info.This(), &arg1);
-  if (!SWIG_IsOK(res1)) {
-    SWIG_Error(SWIG_ArgError(res1), " in method '" "getCPtr" "', argument " "1"" of type '" "void *""'");
-  }
-
-  result = (long)arg1;
-  jsresult = Napi::Number::New(env, result);
-
-  return jsresult;
-#ifndef NAPI_CPP_EXCEPTIONS
-  goto fail;
-fail:
-  return Napi::Value();
-#endif
-}
-
 /* ---------------------------------------------------------------------------
  * PackedData object
  * (objects visible to JS that do not have a dedicated wrapper but must preserve type)

